apps/asap-cli/src/scripts/algolia/delete-index.ts

```diff
@@ -0,0 +1,31 @@
+/* istanbul ignore file */
+import algoliasearch from 'algoliasearch';
+
+export type DeleteAlgoliaIndex = {
+  algoliaAppId: string;
+  algoliaCiApiKey: string;
+  indexName: string;
+};
+
+export const deleteAlgoliaIndex = async ({
+  algoliaAppId,
+  algoliaCiApiKey,
+  indexName,
+}: DeleteAlgoliaIndex): Promise<void> => {
+  const client = algoliasearch(algoliaAppId, algoliaCiApiKey);
+  const unlinkIndex = async (name: string) => {
+    const index = client.initIndex(name);
+    return index.setSettings({ replicas: [] }).wait();
+  };
+  const indices = await client.listIndices();
+  indices.items
+    .filter(({ name }) => name.startsWith(indexName))
```

Comment:
Piotr: isn't this a little dangerous? we are removing all indexes that start with a certain name. I guess the name in our case it usually the name of the branch but I would think we should be a little more specific? I guess it's because the replica and the primary index are both named with the same prefix?

Reply:
i guess its fine, we actually had a similar thing before
```sh
yarn workspace @asap-hub/crn-server run algolia deleteindicespattern -a $ALGOLIA_APP_ID -k $ALGOLIA_API_KEY -r "^$ALGOLIA_INDEX$" -x false
```